build.sbt

```diff
@@ -47,7 +48,8 @@ lazy val core = crossProject(JSPlatform, JVMPlatform, NativePlatform)
     name := "log4cats-core",
     libraryDependencies ++= Seq(
       "org.typelevel" %%% "cats-core"       % catsV,
-      "org.typelevel" %%% "cats-effect-std" % catsEffectV
+      "org.typelevel" %%% "cats-effect-std" % catsEffectV,
+      "org.typelevel" %%% "cats-mtl"        % catsMtlV
```

Comment:
The drawback of having this functionality in the `core` module. 

I can move this code into `log4cats-extras` module. It could be handy if we want to provide some 'non-trivial' functionality: invocation positions (#525), etc.

Reply:
Just an idea, maybe it's worth creating a `log4cats-mtl` module then?

Reply:
Valid point